tests/release_tests/test_launch/test_kubernetes.py

```diff
@@ -0,0 +1,162 @@
+import os
+
+import pytest
+from utils import (
+    cleanup_deployment,
+    create_wandb_and_aws_secrets,
+    get_wandb_api_key,
+    run_cmd,
+    run_cmd_async,
+    setup_cleanup_on_exit,
+    wait_for_image_job_completion,
+)
+from wandb.sdk.launch.launch_add import launch_add
+
+NAMESPACE = "wandb-release-testing"
+ENTITY = "launch-release-testing"
+PROJECT = "release-testing"
+QUEUE = "kubernetes-queue"
+JOB_NAME = "sample_job:v0"  # simple job that counts to 50
+
+LAUNCH_JOB_CONFIG = {
```

Comment:
These environment variables should be automatically injected by the agent. None of these should be required for release testing